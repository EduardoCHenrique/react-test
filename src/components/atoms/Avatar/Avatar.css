.avatar.small {
  width: 20px;
  height: 20px;
}

.avatar.medium {
  width: 100px;
  height: 100px;
}