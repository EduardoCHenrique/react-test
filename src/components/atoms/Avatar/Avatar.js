import React from 'react'
import PropTypes from 'prop-types'
import classNames from 'classnames'
import './Avatar.css';

const Avatar = ({image, className, size}) => {
  return <img src={image} className={classNames(`avatar ${size}`, className)} />
}

Avatar.propTypes = {
  image: PropTypes.string,
  className: PropTypes.string,
  size: PropTypes.oneOf(['small', 'medium']),
}

Avatar.defaultProps = {
  size: 'small'
}

export default Avatar