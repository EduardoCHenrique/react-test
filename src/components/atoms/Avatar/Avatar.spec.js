import React from 'react';
import { shallow } from 'enzyme';
import Avatar from './Avatar.js';

const default_props = {
  image: 'umageurl'
};

function setup(props = default_props) {
  const wrapper = shallow(<Avatar {...props}/>);
  return { wrapper, props };
}

describe('Avatar Test Suite', () => {
  it('Should match snapshot', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});