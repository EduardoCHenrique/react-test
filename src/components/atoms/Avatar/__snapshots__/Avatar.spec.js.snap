// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar Test Suite Should match snapshot 1`] = `ShallowWrapper {}`;
